catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorSensorListTest.kt

```diff
@@ -162,6 +170,13 @@ class FormulaEditorSensorListTest(
             )
 
         onCategoryList().performSelect(sensorName)
+
+        if (isGpsSensor) {
```

Comment:
Please add a `GrantPermissionRule` to this test. Since this test now accesses the user's location it will need additional permissions. Right now a permission dialog appears, probably blocking the execution of subsequent tests.